start-watch/main.js

```diff
@@ -0,0 +1,55 @@
+var timer = {
```

Comment:
You should just store the value as seconds and worry about hours and whatnot in the display function.